src/ast/query.rs

```diff
@@ -148,6 +150,7 @@ impl fmt::Display for SetOperator {
 pub enum SetQuantifier {
     All,
     Distinct,
+    ByName(bool),
```

Comment:
I suggest making two different enums so it is more self-describing: `ByName` and `AllByName` but I don't it is required. If we don't add different enums then I think we should at least add some comments:

```suggestion
    /// if true, `ALL BY NAME`, if false `BY NAME`
    ByName(bool),
```